test/unit/hello.world.controller.test.ts

```diff
@@ -0,0 +1,72 @@
+import { HelloWorldController } from "../../src/controllers/hello-world";
+import { UserProfile } from "@loopback/authentication";
+import { expect, sinon } from "@loopback/testlab";
+import {
+  DefaultCrudRepository,
+  DataSourceConstructor
+} from "@loopback/repository";
+import { UserRepository } from "../../src/repositories/user.repository";
+
+describe("HelloWorldController", () => {
+  let repository: UserRepository;
+  let controller: HelloWorldController;
+  let testUserProfile: UserProfile = {
+    id: "1",
+    name: "foo",
+    email: "foo@bar.com"
+  };
+
+  beforeEach(givenController);
+
+  context("helloWorld()", () => {
+    it("returns Hello World message for a given user", async () => {
+      let stub = repository.create as sinon.SinonStub;
+      const res = await controller.helloWorld("Adam");
+      expect.ok(repository.create, "create was called");
```

Comment:
`repository.create` is a function, it will be always truthy, even when the controller does not call it. I think you should use a different assertion?